### ISSUE TITLE
Switch to BaseExceptionGroup

### ISSUE THREAD

**Issue by macalinao** [pgjones/quart-trio#10]
I keep getting this warning in the console:

```
lib/python3.11/site-packages/quart_trio/app.py:165: TrioDeprecationWarning: trio.MultiError is deprecated since Trio 0.22.0; use BaseExceptionGroup (on Python 3.11 and later) or exceptiongroup.BaseExceptionGroup (earlier versions) instead (https://github.com/python-trio/trio/issues/2211)
```

Would appreciate a fix!

**Comment by Carreau** [comment]
This has been done as part of https://github.com/pgjones/quart-trio/pull/6 I believe, but just not released yet.

**Comment by pgjones** [comment]
Released as 0.11.0

**Comment by Carreau** [comment]
Thank you, your work is always much appreciated !
